<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preprocessingPapersWindow</class>
 <widget class="QMainWindow" name="preprocessingPapersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>预处理试卷</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEditPreprocessingFolder">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChoiceFolder">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>选择试卷文件夹</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditPreprocessingFileList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>741</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartHandle">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveResult">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存结果</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBarPreprocessing"/>
 </widget>
 <resources/>
 <connections/>
</ui>
